src/pages/Transfer/CrossChainTransferForm/components/SelectChainInput/ChainInput.style.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+import { ChevronDown } from '@/assets/icons';
+import { Flex } from '@/component-library/Flex';
+import { List } from '@/component-library/List';
+import { Span } from '@/component-library/Text';
+import { theme } from '@/component-library/theme';
+
+type StyledClickableProps = {
+  $isClickable: boolean;
+};
+
+type StyledListItemSelectedLabelProps = {
+  $isSelected: boolean;
+};
+
+const StyledChain = styled.span`
+  font-size: ${theme.text.s};
+  color: ${theme.colors.textPrimary};
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const StyledChainSelect = styled(Flex)<StyledClickableProps>`
+  background-color: ${theme.tokenInput.endAdornment.bg};
```

Comment:
@danielsimao I'd suggest making these tokenInput properties generic (i.e. `iconInput` or similar) but I'd prefer to do that in a separate PR as we'll have to make changes to other components.

Reply:
I'll add a ticket for this—again, not critical but very quick to fix so we may as well do it sooner rather than later